### PR TITLE
Added scrub finance adapter

### PR DESCRIPTION
**Pull request by g1nt0ki**
##### Logo (High resolution, preferably in .svg and .png, for application on both white and black backgrounds. Will be shown with rounded borders):
https://scrub.finance/lion-logo.png

##### Coingecko ID (so your TVL can appear on Coingecko): (https://api.coingecko.com/api/v3/coins/list)
https://www.coingecko.com/en/coins/lion-scrub-finance

##### Coinmarketcap ID (so your TVL can appear on Coinmarketcap): (https://api.coinmarketcap.com/data-api/v3/map/all?listing_status=active,inactive,untracked&start=1&limit=10000)
https://coinmarketcap.com/currencies/lion-scrub-finance/

##### Short Description (to be shown on DefiLlama):
Scrub Finance is the first deflationary algorithmic token fork pegged to $SVN on the Cronos network with innovative new features!

##### Token address and ticker if any:
LION: 0x49fB98F9b4a3183Cd88e7a115144fdf00fa6fB95

##### Category (full list at https://defillama.com/categories) *Please choose only one:
staking/yield

##### Oracle used (Chainlink/Band/API3/TWAP or any other that you are using):
TWAP

##### forkedFrom (Does your project originate from another project):
tombfinance

##### methodology (what is being counted as tvl, how is tvl being calculated):
amount staked on the platform + dao treasury

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/scrub/index.js exports TVL: 
        
 
 
    cronos                    0
    cronos-staking            23.18 M
    cronos-pool2              55.45 M
    cronos-treasury           3.970201706261447
    staking                   23.18 M
    pool2                     55.45 M
    treasury                  3.970201706261447
    
    total                    0

File: projects/scrub/index.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  await sumTokens(balances, [
+    ['0xD440433dAA33b3e3f2b5421046EAf84bEe6F40D0', rewardPool,], // LION-SVN LP
+    ['0xaDeC6aaAa0765472EE9eBe524BD3454Fd733BAB9', rewardPool,], // TIGER-SVN LP
+  ], block, chain, undefined, { resolveLP: true })
+
+  return balances
+}
+
+module.exports = {
+  misrepresentedTokens: true,
```
Comment (0xngmi) [inline projects/scrub/index.js:22]: i think here tokens are not misrepresented? i might have missed something tho, apologies if so

Comment (tomdoeverything) [reply projects/scrub/index.js:22]: I created PR again
Please remove this PR. thanks

https://github.com/DefiLlama/DefiLlama-Adapters/pull/2267

Comment (tomdoeverything) [reply projects/scrub/index.js:22]: I created PR again
Please remove this PR. thanks

https://github.com/DefiLlama/DefiLlama-Adapters/pull/2267

**Comment by llamatester** [comment]
The adapter at projects/scrub/index.js exports TVL: 
        
 
 
    cronos                    0
    cronos-staking            27.56 M
    cronos-treasury           1.2973370352096878
    cronos-pool2              56.95 M
    staking                   27.56 M
    treasury                  1.2973370352096878
    pool2                     56.95 M
    
    total                    0

**Comment by llamatester** [comment]
The adapter at projects/scrub/index.js exports TVL: 
        
 
 
    cronos                    0
    cronos-staking            24.81 M
    cronos-treasury           1.0959998216106637
    cronos-pool2              50.43 M
    staking                   24.81 M
    treasury                  1.0959998216106637
    pool2                     50.43 M
    
    total                    0

**Comment by llamatester** [comment]
Error while running adapter at projects/scrub/index.js: 
        
 
  
    
    
    Error: Protocol doesn't have total tvl, make sure to export a tvl key either on the main object or in one of the chains
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:206:11
        at processTicksAndRejections (node:internal/process/task_queues:96:5)

**Comment by llamatester** [comment]
The adapter at projects/scrub/index.js exports TVL: 
        
 
 
    cronos                    0
    cronos-staking            22.24 M
    cronos-pool2              48.93 M
    staking                   22.24 M
    pool2                     48.93 M
    
    total                    0

**Pull request merged**